### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by Tykam**
Adding the base routes for the slack commands and the deploy-info command works.

### CODE REVIEW

File: slack-hooks.py [hunk 1]
```diff
@@ -17,5 +28,63 @@ def test_hook():
     return jsonify(response)
 
 
+@app.route('/deploy', methods=['POST'])
+def slack_deploy():
+    response = {
+        'text': 'Deploying'
```
Comment (seth-gordon) [inline slack-hooks.py]: the next step is to replace this stub text with calls to the appropriate methods in `fabric_wrapper.py`, right?

Comment (cbianchi-goscoutgo) [reply slack-hooks.py]: Yup!

File: slack-hooks.py [hunk 2]
```diff
@@ -17,5 +28,63 @@ def test_hook():
...
+
+@app.route('/worker', methods=['POST'])
+def slack_worker():
+    response = {
+        'text': 'Worker'
+    }
+    return jsonify(response)
+
+
+@app.route('/deployment-info', methods=['GET'])
```
Comment (cbianchi-goscoutgo) [inline slack-hooks.py]: This needs to be `POST`.

File: slack-hooks.py [hunk 3]
```diff
@@ -17,5 +28,63 @@ def test_hook():
...
+def slack_deployment_info():
+    """
+    Fetches connect and tpx deployment info for a given environment.
+    """
+    # TODO: Need to add error handling in case the environment given doesn't
+    # exist in our dict.
+    text = request.text.trim()
```
Comment (cbianchi-goscoutgo) [inline slack-hooks.py]: There is no attribute `text`; I think you're looking for `request.form['text']`.

File: slack-hooks.py [hunk 4]
```diff
@@ -17,5 +28,63 @@ def test_hook():
...
+    environment = text.split(' ')[0].lower()
+
+    connect = requests.get(ENVIRONMENT_URLS[environment][0])
+    tpx = requests.get(ENVIRONMENT_URLS[environment][1])
+    info = {'connect': json.loads(connect.text),
+            'tpx': json.loads(tpx.text)
+            }
```
Comment (cbianchi-goscoutgo) [inline slack-hooks.py]: This is a good opportunity to use the slack message attachments but we can worry about pretty formatting later.

Comment (Tykam) [reply slack-hooks.py]: Yeah I figured Skip would be touching all this stuff so I just grabbed something that would let me know i got what we wanted.

**Pull request merged**